src/main/java/com/commercetools/sync/categories/CategorySync.java

```diff
@@ -79,32 +81,33 @@ public CategorySync(@Nonnull final CategorySyncOptions syncOptions) {
      * If a category exists, this category is synced to be the same as the new category draft in this list. If no
      * category exist with such external id, a new category, identical to this new category draft is created.
      *
-     * <p>The {@code statistics} instance is updated accordingly to whether the CTP request was carried out
+     * <p>The returned builder instance is updated accordingly to whether the CTP request was carried out
      * successfully or not. If an exception was thrown on executing the request to CTP, the optional error callback
      * specified in the {@code syncOptions} is called.
      *
      * @param categoryDrafts the list of new category drafts to sync to the CTP project.
      * @return an instance of {@link CompletionStage&lt;U&gt;} which contains as a result an instance of
-     *      {@link CategorySyncStatistics} representing the {@code statistics} instance attribute of
-     *      {@link this} {@link CategorySync}.
+     *      {@link CategorySyncStatisticsBuilder} representing the builder of {@code statistics} baked for
```

Comment:
I meant something like it was not created per sync instance but per invocation of sync method.
Here I adjusted e04195b6ad76160582dd3c58ff407a9deb4e886d